index.html

```diff
@@ -219,28 +219,23 @@ <h3 style="text-align: center;">Orchestrate Jobs</h2>
 	  <h3>&nbsp;&nbsp;&nbsp;<u>Communicate</u></h3>
 	  
 	  <ol>
-      <p>If you want to get involved, you can start by contributing to the <a href="https://github.com/FINRAOS/Dewy" target="_new">GitHub</a> project or reach out to <a href="mailto:opensource@finra.org?Subject=Inquiry%20on%20herd%20project" target="_top">opensource@finra.org</a>. </p>
-      <p>If you have any questions or discussion topics, please post them on <a href=https://github.com/FINRAOS/herd/issues" target="_new">GitHub Issues</a>.</p>
-		
-		
+      <p>We are actively seeking organizations and individuals that are interested in adopting herd and contributing to the development effort. If you want to get involved, you can start by getting to know herd on the <a href="https://github.com/FINRAOS/herd/wiki" target="_new">wiki</a> or the <a href="https://github.com/FINRAOS/herd" target="_new">GitHub</a> project. Or don't hesitate to reach out to <a href="mailto:opensource@finra.org?Subject=Inquiry%20on%20herd%20project" target="_top">opensource@finra.org</a>. </p>
+      <p>If you have any questions or discussion topics, please post them on <a href="https://github.com/FINRAOS/herd/issues" target="_new">GitHub Issues</a>.</p>
+
+		<p><a href="https://help.github.com/" target="_new"><b>GitHub Help</b></a></p><br>
+
 		</ol>
-		
-		<br>
-		<p>&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;<a href="https://help.github.com/" target="_new"><b>GitHub Help</b></a></p>
-				
-		
 		</div>
 		
-		
+		<!---
+
 
-		
-		
 		<div class="row ">
 		<a style="padding-top: 40px; margin-top: -40px;" name="previous_releases">&nbsp;</a>
 		<div class="alert alert-info"><b><font size="5">Previous Releases</font></b></div>
 		
 		
-		<p><a class="btn btn-success btn-primary" role="button" style="background-color:#97c200;border-color:#99c410;" href="https://github.com/FINRAOS/herd/releases/tag/000" target="_new" onclick="_gaq.push([&#39;_trackEvent&#39;, &#39;Visit&#39;, &#39;herd LANDING to Release000&#39;]);">Release Version 0.9</a></p>
+		<p><a class="btn btn-success btn-primary" role="button" style="background-color:#97c200;border-color:#99c410;" href="https://github.com/FINRAOS/herd/releases/tag/0.1.0" target="_new" onclick="_gaq.push([&#39;_trackEvent&#39;, &#39;Visit&#39;, &#39;herd LANDING to Release 0.1.0&#39;]);">Release Version 0.9</a></p>
```

Comment:
That is in a commented out section, I commented out the previous releases until they are relevant.